webcompat/form.py

```diff
@@ -210,9 +210,13 @@ def build_formdata(form_object):
 
 '''.format(**formdata)
     # Add the image, if there was one.
-    if form_object.get('image_upload') is not None:
-        body += '\n\n![Screenshot of the site issue]({image_url})'.format(
-            image_url=form_object.get('image_upload').get('url'))
+    if form_object.get('image_upload'):
+        img_url = form_object.get('image_upload').get('url')
+        img_url_parts = img_url.rsplit('.', 1)
+        img_url_parts.insert(1, 'thumb.')
+        thumb_url = ''.join(img_url_parts)
+        body += '\n\n[![Screenshot of the site issue]({t})]({i})'.format(
```

Comment:
Can we be more descriptive with the named arguments? Maybe just re-use `img_url` and `thumb_url` or `full` and `thumb` (over `t` and `i`)?

edit: just saw you use `thumb_url` and `url` down in bugform.js. Those might be good candidates. 👍